Formula/l/lgeneral.rb

```diff
@@ -46,6 +45,7 @@ def caveats
     <<~EOS
       Requires pg-data.tar.gz or the original DOS version of Panzer General. Can be downloaded from
       https://sourceforge.net/projects/lgeneral/files/lgeneral-data/pg-data.tar.gz/download
+      Compiled without sound and music
```

Comment:
What is this for?

Reply:
To let user know that the game has no sound and music (brew info). This message is printed by the game during startup anyway.

Reply:
Maybe, `sdl2_mixer` is added to compile with sound but it isn't. Yes, user friendly is good.